README.md

```diff
@@ -1 +1,16 @@
-# standard
\ No newline at end of file
+# standard
+# Contributing
+
+Merging a PR in this repository (or pushing a commit to `main`) will automatically build and publish a new version
```

Comment:
This line says `main` but the action config above applies to the `master` branch. Which is it?

Reply:
`master`. Bad copypasta